packages/atti-components/src/helpers/cellItem.js

```diff
@@ -1,39 +1,43 @@
-import { css } from 'styled-components'
+import {css} from 'styled-components'
 
-import { join, map, values, first, last, get } from 'lodash'
+import {join, map, values, first, last, get} from 'lodash'
 
 import wrapMediaQuery from './generators/wrapMediaQuery'
 
 const interpolateMargingCols = cols => {
-  const margingColsCount = Math.floor(cols / 2)
+  const margingColsCount = Math.floor(cols / 2);
   return cols + margingColsCount
 }
 
-const getColumsStyle = (start, end) => `grid-column: ${start} / ${end};`
+const getColumsStyle = (start, end) => `
+  display: block;
+  grid-column: ${start} / ${end};
+`
 const getColumsStyleForIE = (start, end) => `
   -ms-grid-column: ${interpolateMargingCols(start)};
   -ms-grid-column-span: ${interpolateMargingCols(end - start)};
 `
 
+const hideCell = () => "display: none;"
```

Comment:
No need for a function: 
```jsx
const hiddenStyle = css`display: none;`
```

You can then use it as a reference inside styled`` scope:

```jsx
const MyCell = styled`
  ${hiddenStyle};
`
```

Reply:
Done